### PR TITLE
PR for #237

### PR DESCRIPTION
**Pull request by hirako2000**
**This PR is related to issue #237**

### CODE REVIEW

File: src/components/meshes/Extrude.js [hunk 1]
```diff
@@ -39,12 +39,12 @@ class Extrude extends MeshComponent {
...
   buildGeometry(params = {}) {
-    const GConstruct = params.buffer && !params.softbody ? ExtrudeBufferGeometry : ExtrudeGeometry;
-
-    return new GConstruct(
+    const extrudeGeometry = new ExtrudeGeometry(
       params.geometry.shapes,
       params.geometry.options
     );
+
+    return params.buffer && !params.softbody ? new BufferGeometry().fromGeometry(extrudeGeometry) : extrudeGeometry;
```
Comment (sasha240100) [inline src/components/meshes/Extrude.js]: `params.softbody` should not be used. Physics is now completely separate from whs. Can you remove it, please?

-----

Good idea to use `BufferGeometry().from`. We need to do it also for other meshes. I'll make an issue.

Comment (sasha240100) [inline src/components/meshes/Extrude.js]: https://github.com/WhitestormJS/whitestorm.js/issues/239

Comment (hirako2000) [inline src/components/meshes/Extrude.js]: Sure, will remove the softbody property from the extrude component.

**Comment by sasha240100** [comment]
@hirako2000 how it goes?

**Comment by hirako2000** [comment]
Removed softbody support.

**Pull request merged**